orientdb/src/main/java/com/yahoo/ycsb/db/OrientDBClient.java

```diff
@@ -17,197 +17,145 @@
 
 package com.yahoo.ycsb.db;
 
-import com.orientechnologies.common.exception.OException;
-import com.orientechnologies.orient.client.remote.OEngineRemote;
-import com.orientechnologies.orient.client.remote.OServerAdmin;
-import com.orientechnologies.orient.core.db.ODatabaseRecordThreadLocal;
+import com.orientechnologies.orient.core.config.OGlobalConfiguration;
+import com.orientechnologies.orient.core.db.OPartitionedDatabasePool;
 import com.orientechnologies.orient.core.db.document.ODatabaseDocumentTx;
-import com.orientechnologies.orient.core.db.record.OIdentifiable;
 import com.orientechnologies.orient.core.dictionary.ODictionary;
-import com.orientechnologies.orient.core.exception.ODatabaseException;
+import com.orientechnologies.orient.core.exception.OConcurrentModificationException;
 import com.orientechnologies.orient.core.index.OIndexCursor;
-import com.orientechnologies.orient.core.intent.OIntentMassiveInsert;
-import com.orientechnologies.orient.core.intent.OIntentMassiveRead;
-import com.orientechnologies.orient.core.intent.OIntentNoCache;
 import com.orientechnologies.orient.core.record.ORecord;
 import com.orientechnologies.orient.core.record.impl.ODocument;
-import com.yahoo.ycsb.ByteIterator;
-import com.yahoo.ycsb.DB;
-import com.yahoo.ycsb.DBException;
-import com.yahoo.ycsb.Status;
-import com.yahoo.ycsb.StringByteIterator;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-
-import java.io.IOException;
-import java.util.HashMap;
-import java.util.Map.Entry;
-import java.util.Properties;
-import java.util.Set;
-import java.util.Vector;
+import com.yahoo.ycsb.*;
+
+import java.io.File;
+import java.util.*;
+import java.util.concurrent.locks.Lock;
+import java.util.concurrent.locks.ReentrantLock;
 
 /**
  * OrientDB client for YCSB framework.
+ * <p>
+ * Properties to set:
+ * <p>
+ * orientdb.url=local:C:/temp/databases or remote:localhost:2424 <br>
+ * orientdb.database=ycsb <br>
+ * orientdb.user=admin <br>
+ * orientdb.password=admin <br>
+ *
+ * @author Luca Garulli
  */
 public class OrientDBClient extends DB {
-
   private static final String CLASS = "usertable";
-  private ODatabaseDocumentTx db;
-  private ODictionary<ORecord> dictionary;
-  private boolean isRemote = false;
 
-  private static final String URL_PROPERTY = "orientdb.url";
```

Comment:
Sorry my comment wasn't clear. The format of the entire class changed. The changes removed all the \*\_PROPERTY* variables and instead inlined all the values. This makes it hard to review. It is typically better to stick with the existing code format if possible.